src/MeetingsSDKAdapter.js

```diff
@@ -895,6 +904,70 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     return state$;
   }
 
+  /**
+   * Attempts to toggle settings to the given meeting ID.
+   * A settings toggle event is dispatched.
+   *
+   * @private
+   * @param {string} ID ID of the meeting to toggle settings
+   */
+  handleSettings(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+    const showSettings = !this.meetings[ID].showSettings;
+
+    this.meetings[ID].showSettings = showSettings;
+
+    sdkMeeting.emit(EVENT_SETTINGS_TOGGLE, {
```

Comment:
maybe TOGGLED would have been more semantic. We can discuss, no need to fix this now, other events may also need to be renamed in this spirit.